lib/util/determine-files.js

```diff
@@ -0,0 +1,81 @@
+let path = require("path");
+let fs = require("fs");
+
+/* Returns a function that determines files to process
```

Comment:
a) "returns" ( 🙅‍♂️ sentence case)
b) "to process" is irrelevant here
c) "determines files within a directory, optionally filtered by `predicate`"? (see below)